bin/react-scripts.js

```diff
@@ -5,14 +5,17 @@ var args = process.argv.slice(3);
 
 switch (script) {
 case 'build':
-case 'start':
 case 'eject':
+case 'start':
   spawn(
     'node',
     [require.resolve('../scripts/' + script)].concat(args),
     {stdio: 'inherit'}
   );
   break;
+case 'test':
+  require('jest/bin/jest');
```

Comment:
yeah. People can also `npm install -g jest` globally and then run `jest -u`. That's usually the most convenient option because you can run it from any folder and Jest traverses up the tree until it finds a config.